sktime/utils/mlflow_sktime.py

```diff
@@ -0,0 +1,534 @@
+        mlflow.models.Model configuration to which to add the python_function flavor.
+    signature : mlflow.models.signature.ModelSignature, optional (default=None)
+        Model Signature mlflow.models.ModelSignature describes
+        model input and output :py:class:`Schema <mlflow.types.Schema>`. The model
+        signature can be :py:func:`inferred <mlflow.models.infer_signature>` from
+        datasets with valid model input (e.g. the training dataset with target column
+        omitted) and valid model output (e.g. model predictions generated on the
+        training dataset), for example:
+
+        .. code-block:: py
+
+          from mlflow.models.signature import infer_signature
+          train = df.drop_column("target_label")
+          predictions = ... # compute model predictions
+          signature = infer_signature(train, predictions)
+
+        .. Warning:: if performing probabilistic forecasts (``predict_interval``,
+          ``predict_quantiles``) with a sktime model, the signature
+          on the returned prediction object will not be correctly inferred due
+          to the Pandas MultiIndex column type when using the these methods.
+          ``infer_schema`` will function correctly if using the ``pyfunc`` flavor
+          of the model, though.
+    input_example : Union[pandas.core.frame.DataFrame, numpy.ndarray, dict, list, csr_matrix, csc_matrix], optional (default=None)
+        Input example provides one or several instances of valid model input.
+        The example can be used as a hint of what data to feed the model. The given
+        example will be converted to a ``Pandas DataFrame`` and then serialized to json
+        using the ``Pandas`` split-oriented format. Bytes are base64-encoded.
+    pip_requirements : Union[Iterable, str], optional (default=None)
+        Either an iterable of pip requirement strings
+        (e.g. ["sktime", "-r requirements.txt", "-c constraints.txt"]) or the string
+        path to a pip requirements file on the local filesystem (e.g. "requirements.txt")
+    extra_pip_requirements : Union[Iterable, str], optional (default=None)
+        Either an iterable of pip requirement strings
+        (e.g. ["pandas", "-r requirements.txt", "-c constraints.txt"]) or the string
+        path to a pip requirements file on the local filesystem (e.g. "requirements.txt")
+
+
+    See Also
+    --------
+    MLflow
+
+    References
+    ----------
+    .. [1] https://www.mlflow.org/docs/latest/python_api/mlflow.models.html#mlflow.models.Model.save
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.arima import ARIMA
+    >>> from sktime.utils import mlflow_sktime
+    >>> y = load_airline()
+    >>> forecaster = ARIMA(  # doctest: +SKIP
+    ...     order=(1, 1, 0),
+    ...     seasonal_order=(0, 1, 0, 12),
+    ...     suppress_warnings=True)
+    >>> forecaster.fit(y)  # doctest: +SKIP
```

Comment:
Just based on what I read in the docs:  

_If including docstring examples that use soft dependencies, ensure to skip doctest. To do this add a `# doctest: +SKIP` to the end of each line in the doctest to skip._ (https://www.sktime.org/en/stable/developer_guide/dependencies.html?#adding-a-soft-dependency)

Just let me know in case I have some misunderstanding here!

Resolving this for now